iCubGenova04/wrappers/MAIS/left_arm-mais_wrapper_nohands.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+
+    <device xmlns:xi="http://www.w3.org/2001/XInclude" name="left_arm-mais_wrapper" type="analogServer">
+        <param name="period">       10                         </param>
+        <param name="name">       /icub/left_hand/analog:o      </param>
+        
+        <action phase="startup" level="5" type="attach">
+            <paramlist name="networks">
+                <elem name="leftMais"> left_arm-eb26-j12_15-mais </elem>
+            </paramlist>
+        </action>
+
+        <action phase="shutdown" level="5" type="detach" />
+        </device>
+
```

Comment:
Unless I am wrong, this file and the corresponding file for the right are not used anywhere, so they can be removed?